db/migrate/20241015103928_add_user_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToArticles < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :articles, :user, null: false, foreign_key: true
```

Comment:
Если ты на уровне модели ActiveRecord сделал колбек что если юзер будет удален то и все его посты будут удалены, предлагаю также сделать это ограничение тут

```suggestion
    add_reference :articles, :user, null: false, foreign_key: { on_delete: :cascade }
```